phpmyfaq/src/phpMyFAQ/Controller/Frontend/UserController.php

```diff
@@ -251,4 +250,31 @@ public function requestUserRemoval(Request $request): JsonResponse
             return $this->json(['error' => Translation::get('err_sendMail')], Response::HTTP_BAD_REQUEST);
         }
     }
+
+    /**
+     * @throws Exception
+     */
+    #[Route('api/user/remove-twofactor', methods: ['POST'])]
+    public function removeTwofactorConfig(Request $request): JsonResponse
+    {
+        $data = json_decode($request->getContent());
```

Comment:
Please add a check if the user is registered.

Reply:
Not necessary at this time, because we only use CurrentUser.